src/TraceEvent/Parsers/GCDynamicData.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Microsoft.Diagnostics.Tracing.Parsers.GCDynamicData
+{
```

Comment:
we probably don't need to have this as a separate file, it can be folded into the event parser src

Reply:
I had separated this into a separate file in my prototype.  You're right that it could live in the CLR parser.  My rationale for keeping it in a separate file is that there's enough complexity and "specialness" to this that I felt it warranted some separation.

Reply:
Sorry - I thought this referred to GCDynamicTraceEventParser.cs.  I don't think this file is used at all and you can safely delete it.